submissions/AnnaStoyano/memory-pair-game/index.js

```diff
@@ -0,0 +1,152 @@
+class Game {
+    constructor(cards) {
+        this.container = document.querySelector('.container');
+        this.cards = cards;
+    }
+
+    render() {
+        this.cards.forEach(card => this.container.insertAdjacentElement('beforeend', card.render()));
+    }
+
+    checkEqualCards(activeCards) {
+        const activeCardsNames = activeCards.map(card => card.cardForDisplay.dataset.name);
+        if (activeCardsNames[0] === activeCardsNames[1]) {
+            activeCards.forEach(card => card.addOpened());
+            return true;
+        }
+        return false;
+    }
+
+    removeActiveCards(cards, isOpened) {
+        let time = isOpened ? 0 : 600;
+        setTimeout(function () {
```

Comment:
It's not a blocker, but you can use arrow functions for simplifying this expression
```suggestion
        setTimeout(() => cards.forEach(card => card.removeActive()), time)
```